### PR TITLE
Working notebook explaining how to build MDNs with Edward

### PR DESCRIPTION
**Pull request by cbonnett**
This is slight adapted version of the  notebook in https://github.com/cbonnett/MDN_Edward_Keras_TF
The notebook explains how to make a Mixture Density Network in Edward using Keras and TensorFlow.
It is relevant to #35

### CODE REVIEW

**Comment by dustinvtran** [comment]
Awesome. Can you run the ipython notebook and save it with all the plotted figures displayed? (Some people may not actually run the notebook but will just read it.) Also, can you fix some of the grammatical errors?

**Comment by cbonnett** [comment]
All plots are there now , and hopefully all typos and grammar errors are gone (was never my forte...).

**Comment by dustinvtran** [comment]
No problem re:pull request format. It's structure I personally like although it's not necessary unless it's a big PR. Thanks again for the notebook.

**Comment by raddy** [comment]
Debugging nans still -- is it possible to do check_numeric() calls in the loss calculation or will these prohibit the auto-gradient generation?
